contracts/TransferLimitModule.sol

```diff
@@ -171,6 +170,14 @@ contract TransferLimitModule is Module, SignatureDecoder, SecuredTokenTransfer {
         // Increase nonce and execute transaction.
         nonce++;
 
+        // If time period is over, reset expenditure.
+        TransferLimit storage transferLimit = transferLimits[token];
```

Comment:
Why to we retrieve the transferLimit here, before we know if it is required. Every storage access costs 200 gas, so that is quite expensive